doc/classes/RenderingServer.xml

```diff
@@ -515,6 +515,14 @@
 				Sets the index for the [CanvasItem].
 			</description>
 		</method>
+		<method name="canvas_item_set_inherit_visibility_layer">
+			<return type="void" />
+			<param index="0" name="item" type="RID" />
+			<param index="1" name="enabled" type="bool" />
+			<description>
+				If [param enabled] is [code]true[/code], the [CanvasItem] will inherit its parent's visibility layers.
```

Comment:
Gaaah, I hate the sheer inconsistency of all **Server** classes and it always has me conflicted.
On one hand, explicitly calling it **CanvasItem** makes it considerably easier to think about, but on the other hand, it's technically incorrect and misleading at worst.
A decent middle-ground is to write as follows:
```suggestion
				If [param enabled] is [code]true[/code], the given canvas item inherits its parent's visibility layers. Equivalent to [member CanvasItem.inherit_visibility_layer].
```